doc/src/test.md

```diff
@@ -2,7 +2,7 @@
 
 ```
 Usage: hevm test [--root STRING] [--project-type PROJECTTYPE] [--rpc TEXT]
-                 [--number W256] [--verbose INT] [--coverage] [--match STRING]
+                 [--number W256] [--coverage] [--match STRING]
```

Comment:
Should `verb` be added here? Or why is that not necessary?

Reply:
I'm actually thinking about removing this part entirely. It's trivial to get via `--help` and it's likely also to be more accurate. What do you think? I wanna do that as a follow-up PR. I don't think this reference is very helpful. The tutorial, and other parts are way more useful. The value-add here is minimal. I have seen people navigate to this and then be confused how it's helpful....

Reply:
I have now written a PR to significantly improve our documentation, here #690